tests/functional/pages.php

```diff
@@ -0,0 +1,65 @@
+<?php
+define('INSTALL_ROOT',  realpath(__DIR__ . '/../../') . '/');
+
+// We always work with UTF8 encoding
+mb_internal_encoding('UTF-8');
+
+// Make sure we have a timezone set
+date_default_timezone_set('Europe/Paris');
+
+require __DIR__ . '/../../vendor/autoload.php';
+
+// Set an environment variable so that the instance will use content from test files
+putenv("AUTOMATED_TESTS=true");
+
+// Launch PHP dev server in the background
+chdir(INSTALL_ROOT);
+exec('./start.sh -remote > /dev/null 2>&1 & echo $!', $output);
+
+// We will need the pid to kill it, beware, this is the pid of the bash process started with start.sh
+$pid = $output[0];
+
+// Pause to let time for the dev server to launch in the background
+sleep(3);
+
+$paths = [
+    ['', 200, 'You are looking at', 'Type your search here'],
+    ['3locales/', 200, 'One source locale', 'Type your search here'],
+    ['accesskeys/', 200, 'Check your access keys', 'Label entity'],
+    ['changelog.php/', 200, 'You are looking at', 'Type your search here'],
```

Comment:
I think we should remove this, and also remove it from urls.php (I'll file a new issue for that).
It's probably a remnant of the past (before nice URLs), and it also generate warnings